artifactory/utils/prompt/promptutils.go

```diff
@@ -174,3 +179,59 @@ func GetRepositories(resolveRes *viper.Viper, repoTypes ...utils.RepoType) ([]st
 
 	return utils.GetRepositories(artAuth, repoTypes...)
 }
+
+func CreateBuildConfig(global bool, confType utils.ProjectType) error {
```

Comment:
I suggest creating npmconfig.go file and put this function there, as do other techs - go, pip, gradle and maven.

Reply:
Done.